main/src/cgeo/geocaching/CacheDetailActivity.java

```diff
@@ -1934,6 +1934,7 @@ public static void startActivity(final Context context, final String geocode, fi
     }
 
     private ActionMode mActionMode = null;
+    private boolean mSelectionModeActive = false;
```

Comment:
We don't usually follow the convention `mXXX` for member fields. I'd prefer that we stay consistent and use `selectionModeActive` instead.